src/args.ts

```diff
@@ -96,4 +96,14 @@ export let args : ArgDescriptor[] = [
     description: 'Path to h2-push manifest',
     type: String,
   },
+  {
+    name: 'proxy-root',
+    description: 'Url to proxy from',
+    type: String,
+  },
+  {
+    name: 'proxy-target',
+    description: 'Url to proxy to',
```

Comment:
A little more detail here: full URL or a path? reference that this is related to proxy-target?